xbmc/video/dialogs/GUIDialogAudioSubtitleSettings.cpp

```diff
@@ -156,6 +158,16 @@ void CGUIDialogAudioSubtitleSettings::OnSettingChanged(const CSetting *setting)
       g_application.m_pPlayer->SetAudioStream(m_audioStream);    // Set the audio stream to the one selected
     }
   }
+  else if (settingId == SETTING_VIDEO_STREAM)
```

Comment:
So a user would select a different stream for quality reasons, right? The other video and quality related settings are in video settings dialog. I would expect the stream selection there.